### ISSUE TITLE
[Vue.js][select][v-model] select element's binded property doesn't update.

### ISSUE THREAD

**Issue by clemquinones** [Dogfalo/materialize#5565]
I'm having a difficulty on the reactivity of the data property to the select element.

Surely, this function is executed first:
```javascript
$('select').material_select()
// I'm aware that this function will create an additional ul/li(s) elements that uses as front list interface on behalf of the select element.
```
When selecting a choice, it doesn't trigger a changed/updated event to notify the vue that's why the property binded to the element doesn't update.

I have this property on one of my vue component:
```javascript
data() {
 return {
  transaction: { type: null }
  ...
```

And binded it to my select element:
```html
<select v-model="transaction.type" id="transaction-type">
...
</select>
```

On jQuery, I can catch the value using this code but I'd like to know other way that is simplier and more of a vue.js way.
```javascript
$('#transaction-type').on('change', event => {
 let choice = event.target.value
 //
 });
```

Any tips would be greatly appreciated.

**Comment by joseychechu** [comment]
I use this code:

```
     new Vue({
        el: "#vue",
        data: {
            filter: 'default_value',
        },
        methods: {
            //
        },
        watch:{
            filter: function(){
                //Function
            },
        },
        mounted: function(){
            //On Change for filter
            $("#filter").change(function(){
                this.filter = $("#filter").val();
            }.bind(this));
        }
    });

```

**Comment by Dogfalo** [comment]
Should be fixed in 1.0 since we have an event listener that watches for the change event on the original select, but there is no good way that I know of to resolve these kinds of issues with the various JS frameworks as they vary slightly in implementation.

The best solution is to have small bits of code as @joseychechu suggested to bind the two properly
